docs/source/technical_tutorials/working_with_r/cmr_search_in_r.ipynb

```diff
@@ -15,7 +15,7 @@
    "source": [
```

Comment:
<p>We use federated search here because <code>maap-py</code> only covers NASA Earthdata - where Federated search adds MAAP, ESA, and VEDA </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>